rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -12793,16 +12767,14 @@ void opcode_042c(const ScriptArguments& args, const ScriptInt arg1) {
 }
 
 /**
-	@brief %2d% = metric_to_imperial %1d%
+	@brief %2d% = round(meters_to_feet(%1d%)) 
 
 	opcode 042d
 	@arg arg1 
 	@arg arg2 
 */
 void opcode_042d(const ScriptArguments& args, const ScriptInt arg1, ScriptInt& arg2) {
-	RW_UNIMPLEMENTED_OPCODE(0x042d);
-	RW_UNUSED(arg1);
-	RW_UNUSED(arg2);
+	arg2 = static_cast<ScriptInt>(std::round(static_cast<ScriptFloat>(arg1) * 3.2808398));
```

Comment:
Same as 0425, the actual behaviour is slightly less accurate: https://www.gtamodding.com/wiki/042D

Reply:
What do you think about floor(arg1 * 3.3333334)?

Reply:
That seems close to the expected behaviour